multi-cluster-service/aws/setup-aws.sh

```diff
@@ -0,0 +1,65 @@
+export CLUSTER_MAIN=${1:-"psmdb-cluster-mcs-setup-test-1"}
+export CLUSTER_REPL=${2:-"psmdb-cluster-mcs-setup-test-2"}
+export CLUSTER_MAIN_CIDR=10.10.0.0/16
+export CLUSTER_REPL_CIDR=10.12.0.0/16
+
+########################## Create cluster 1 ############################
+eksctl create cluster -f eks-cluster-mcs-1.yaml
+
+eksctl utils associate-iam-oidc-provider --region=eu-west-3 --cluster=${CLUSTER_MAIN} --approve
+eksctl create iamserviceaccount --region eu-west-3 --name cloud-map-mcs-controller-manager --namespace cloud-map-mcs-system --cluster ${CLUSTER_MAIN} --attach-policy-arn arn:aws:iam::aws:policy/AWSCloudMapFullAccess --approve --override-existing-serviceaccounts
+
+kubectl create ns mcs && kubens mcs
```

Comment:
Replace `kubens` with `kubectl config set-context --current --namespace <namespace>` which can be used without additional dependencies (or at the start of the script check if the binary exists and exit if not).
I don't have `kubens` ;)